dotcom-rendering/src/web/components/FixedSmallSlowIV.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { breakpoints } from '@guardian/source-foundations';
+import { ContainerLayout } from './ContainerLayout';
+import { FixedSmallSlowIV } from './FixedSmallSlowIV';
+
+import { trails } from '../../../fixtures/manual/trails';
+
+export default {
+	component: FixedSmallSlowIV,
+	title: 'Components/FixedSmallSlowIV',
+	parameters: {
+		chromatic: {
+			viewports: [
+				breakpoints.mobile,
```

Comment:
If we're planning on adding all these breakpoints for every type of container we probably need to find a way to give chromatic some kind of focus so that it only checks files that could have changed.

Reply:
> If we're planning on adding all these breakpoints for every type of container we probably need to find a way to give chromatic some kind of focus so that it only checks files that could have changed.
> 
> This might be some help? https://www.chromatic.com/docs/turbosnap

Interesting. It's hard to say if this is a good option for us or not, I think the write up at the link you gave (thanks) is fairly accurate in being hesitant but if this does work the way we want it to then it could be a big win

Reply:
Raised https://github.com/guardian/dotcom-rendering/pull/4768 for this, tried to explain the incompatibility the way I understand it in this PR